server/src/mindustry/server/ServerControl.java

```diff
@@ -259,7 +259,17 @@ protected void setup(String[] args){
     }
 
     protected void registerCommands(){
-        handler.register("help", "Displays this command list.", arg -> {
+        handler.register("help", "[command]", "Display the command list, or get help for a specific command.", arg -> {
+            if (arg.length > 0){
```

Comment:
incorrect formatting similar to this in several places, see guide

Reply:
Fixed